modules/learn/pages/data/validity.adoc

```diff
@@ -0,0 +1,70 @@
+
+[#atomicity-definition-and-implementation]
+=== Atomicity
+
+_Atomicity_ guarantees that when multiple data-elements need to be modified such that only the successful modification of all justifies the modification of any, _either_ all the modifications do occur successfully; _or_ none of them occurs.
+
+Atomicity is innately supported by the xref:learn:data/document-data-model.adoc[Couchbase Data Model]; which is naturally expressive of hierarchical data, and therefore typically only requires a single document for data that, under a relational model, would require multiple tables; and would therefore maintain data-elements discretely.
+On Couchbase Server, changes made to a single document are all written simultaneously, thereby guaranteeing atomicity of all data-elements within the document for every write.
+
+Additionally, Couchbase Server supports the atomic inserting, updating, and deleting of data-elements across multiple documents, by means of xref:learn:data/transactions.adoc[Distributed ACID Transactions].
+
+[#consistency-definition-and-implementation]
+=== Consistency
+
+_Consistency_ requires that modifications to data occur only in accordance with required equivalencies and invariants.
+For example, multiple copies of a single data-element must all be updated identically, when an update to any one occurs.
+
+Couchbase Server provides xref:learn:clusters-and-availability/intra-cluster-replication.adoc[Intra-Cluster Replication]; which ensures data-availability with up to three replicas, deployed across the cluster.
+The _Database Change Protocol_ (DCP) keeps active and replica data-elements both consistent and up-to-date with _eventual consistency_.
+
+Additionally, Couchbase Server provides xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication (XDCR)]; which uses eventual consistency to replicate data-modifications across different clusters, potentially located in different data centers.
+
+The Couchbase xref:learn:services-and-indexes/services/index-service.adoc[Index Service] provides high-performance data-access; and as such, maintains fast-access copies of specified data.
+Such copies are updated with eventual consistency, across the cluster.
+A xref:n1ql:n1ql-rest-api/index.adoc#table_xmr_grl_lt[Read Your Own Writes] mechanism is available, to allow queries to read modified data from an index at the earliest point at which consistency is achieved.
+
+When data-elements must be updated consistently across multiple documents, xref:learn:data/transactions.adoc[Distributed ACID Transactions] can be used.
+
+[#isolation-definition-and-implementation]
+=== Isolation
+
+_Isolation_ ensures that no transaction's data-modifications are visible to any other transaction prior to commitment: a _transaction_ being an operation that performs an atomic update, involving multiple discrete data-elements.
```

Comment:
Weellll, _Read Committed_ Isolation means this..  but there are much higher isolation levels possible.  Serializable, the highest, means that you can create a theoretical history where every transaction happened in serial.  At its simplest isolation level really means, what's the behaviour of concurrent transactions.

Reply:
So, is the behaviour 'levelled' in terms of how protected are one transaction's changes from those of another? How, I wonder, do we say this.